subscription-service/src/test/resources/ngsild/subscription_with_time_interval_less_than_0.json

```diff
@@ -0,0 +1,29 @@
+{
+  "id":"urn:ngsi-ld:Subscription:1",
+  "type":"Subscription",
+  "entities": [
+    {
+      "type": "Beehive"
+    },
+    { "id": "urn:ngsi-ld:Beehive:1234567890",
+      "type": "Beehive"
+    },
+    {
+      "idPattern": "urn:ngsi-ld:Beehive:1234*",
+      "type": "Beehive"
+    }
+  ],
+  "timeInterval": 0.1,
```

Comment:
"0.1" is not "less than 0" :)

Reply:
Yes I admit that the name is not good for this use case. "les than 1" is better